### DISCUSSION TITLE
Blank project

### DISCUSSION THREAD

**Discussion by picasso2000** [leokhoa/laragon#465] [Q&A]
How to add an BLANK project in separate folders?
Currently it adds new blank project to the root folder.

```
# Options
AutoCreateDatabase=true
Cached=true

# Blank: an empty project
Blank=

# WordPress
WordPress=https://wordpress.org/latest.zip 
```

Is there a possibility that it will not add a database for an empty project?

**Comment by josealgardataborges** [comment]
Why don't you just create the folder you need inside the laragon www folder (usually C:\laragon\www)?

That way you don't have to worry about the creation of the "database" associated with the "blank. an empty project"
